docs/contributing.rst

```diff
@@ -214,6 +214,7 @@ Some details about using Modin on Ray with Asv:
 Some details for maintenance:
 
 * ``modin/asv_bench/asv.conf.json`` contains the modin dependencies, with which testing takes place.
+
```

Comment:
remove this extra line

Reply:
This extra line is needed to fix sphinx warning message in building process.

Reply:
The sentence "We need to keep them up to date according to the dependencies in ``setup.py``." should be related to the sentence "``modin/asv_bench/asv.conf.json`` contains the modin dependencies, with which testing takes place.". Then you should rephrase these sentences to fit into one sentence.

Reply:
Okay, I see.